content/applications/hr/lunch/management.rst

```diff
@@ -0,0 +1,189 @@
+:ref:`confirmed <lunch/confirm-orders>` upon arrival, and :ref:`employees can be notified
+<lunch/notify>`, from the :ref:`Today's Orders <lunch/todays-orders>` dashboard, or the
+:ref:`Control Vendors <lunch/control_vendors>` dashboard.
+
+To manage the *Lunch* app, users need :guilabel:`Administrator` rights, set by navigating to the
+:menuselection:`Settings app` and selecting :guilabel:`Manage Users`. Then, click on the desired
+user to view their access rights.
+
+For more information on access rights, refer to the :doc:`Access rights
+</applications/general/users/access_rights/>` documentation.
+
+.. note::
+   Only users with administration rights are able to view the :guilabel:`Manager` and
+   :guilabel:`Configuration` menus.
+
+.. _lunch/todays-orders:
+
+Today's Orders
+==============
+
+To view and manage the orders for the day, navigate to :menuselection:`Lunch app --> Manager -->
+Today's Orders`. All orders for the day are presented in a list view on the :guilabel:`Today's
+Orders` dashboard, with a filter for :guilabel:`Today`, and grouped by :guilabel:`Vendor`, by
+default.
+
+The following information appears in the list:
+
+- :guilabel:`Order Date`: the date the order was placed.
+- :guilabel:`Vendor`: the vendor the product is being ordered from.
+- :guilabel:`Product`: the specific product ordered.
+- :guilabel:`Extras`: any extras selected for the product.
+- :guilabel:`Notes`: any information needed to be sent to the vendor.
+- :guilabel:`User`: the user who ordered the product.
+- :guilabel:`Lunch Location`: where the product is to be delivered.
+- :guilabel:`Price`: the total price for the product, including all extras.
+- :guilabel:`Status`: the current status of the product.
+- :guilabel:`Company`: the company the order was placed under. This only appears in a multi-company
+  database.
+
+.. image:: management/today.png
+   :align: center
+   :alt: The list that appears in the Today's Orders dashboard, with the filters and top column
+         names highlighted.
+
+.. _lunch/cancel:
+
+Cancel orders
+-------------
+
+All users can cancel an order, not just managers of the *Lunch* app.
+
+To cancel an order from a vendor, individual products can be canceled one at a time.
+
+On the :guilabel:`Today's Orders` dashboard, a :guilabel:`✖️ Cancel` icon is shown at the far-right
+of each product line that can be canceled. Click the :guilabel:`✖️ Cancel` icon to cancel the order
+for that individual product.
+
+.. note::
+   Only products with a red :guilabel:`Status` tag of :guilabel:`Ordered` can be canceled.
+
+.. image:: management/cancel.png
+   :align: center
+   :alt: Order lines with the cancel icon highlighted.
+
+.. _lunch/send-orders:
+
+Send Orders
+-----------
+
+The first step in managing the *Lunch* app is to send the orders to the vendors.
+
+When orders are ready to be sent, click the :guilabel:`Send Orders` button that appears next to each
+vendor's name and phone number.
```

Comment:
a helpful reminder, via some elaboration/clarification, on where the user can *send* orders from the app, may help you paint a more "clear" picture for the reader

Reply:
Ahh yes- NOPE, theyhave to CALL or email or however they do the orders.